My Simple App/Business Logic/InternalGeolocation.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  InternalGeolocation.swift
+//  My Simple App
+//
+//  Created by Erick Sanchez on 8/5/20.
+//  Copyright © 2020 Erick Sanchez. All rights reserved.
+//
+
+import CoreLocation
+
+// Wrapper protocol of CLGeocoder.
+protocol InternalGeocoder {
```

Comment:
woodward4422 2 days ago @woodward4422
Why do we need this as a protocol? Isnt this only ever being implemented one?

LinnierGames 34 minutes ago • 
 Author Owner
Not quite, since this is a protocol I could define a Mock implementation or fake implementation for unit tests to use